build can't find conf.py?

If I try to build circuitpython with current master (dd0f8689a), I get:

```
Use make V=1, make V=2 or set BUILD_VERBOSE similarly in your environment to increase build verbosity.
FREEZE modules
Traceback (most recent call last):
  File "../../py/makeversionhdr.py", line 108, in <module>
    make_version_header(sys.argv[1])
  File "../../py/makeversionhdr.py", line 70, in make_version_header
    info = get_version_info_from_docs_conf()
  File "../../py/makeversionhdr.py", line 55, in get_version_info_from_docs_conf
    with open(os.path.join(os.path.dirname(sys.argv[0]), "..", "docs", "conf.py")) as f:
IOError: [Errno 2] No such file or directory: '../../py/../docs/conf.py'
make: *** [../../py/py.mk:273: build/genhdr/mpversion.h] Error 1
make: *** Waiting for unfinished jobs....
```

And indeed, there is no `py/conf.py`. It looks as if it was relocated in 46e7f8e to the top-level of the repository.

Sorry, duh.  My bad.

I'm building the esp8266 port, from `ports/esp8266`, and I am running `make`.

```
$ pwd
/home/lars/src/micropython/ports/esp8266
$ git log --oneline -1
dd0f8689a (HEAD, circuit/master) Merge pull request #786 from notro/rtc_set_datetime_precision
$ esp-sdk make
Use make V=1, make V=2 or set BUILD_VERBOSE similarly in your environment to increase build verbosity.
Traceback (most recent call last):
  File "../../py/makeversionhdr.py", line 108, in <module>
    make_version_header(sys.argv[1])
  File "../../py/makeversionhdr.py", line 70, in make_version_header
    info = get_version_info_from_docs_conf()
  File "../../py/makeversionhdr.py", line 55, in get_version_info_from_docs_conf
    with open(os.path.join(os.path.dirname(sys.argv[0]), "..", "docs", "conf.py")) as f:
IOError: [Errno 2] No such file or directory: '../../py/../docs/conf.py'
make: *** [../../py/py.mk:273: build/genhdr/mpversion.h] Error 1
```

--- a/py/makeversionhdr.py
+++ b/py/makeversionhdr.py
@@ -52,7 +52,7 @@ def get_version_info_from_git():
     return git_tag, git_hash, ver
 
 def get_version_info_from_docs_conf():
-    with open(os.path.join(os.path.dirname(sys.argv[0]), "..", "docs", "conf.py")) as f:
+    with open(os.path.join(os.path.dirname(sys.argv[0]), "..", "conf.py")) as f:
         for line in f:
             if line.startswith("version = release = '"):
                 ver = line.strip().split(" = ")[2].strip("'")

